tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -1264,6 +1264,112 @@ def test_vpc_config(
             "delete_vpcconfig_get_function_response", delete_vpcconfig_get_function_response
         )
 
+    @markers.aws.validated
+    def test_invalid_vpc_config_subnet(
+        self, create_lambda_function, lambda_su_role, snapshot, aws_client, cleanups
+    ):
+        """
+        Test invalid "VpcConfig.SubnetIds" Property on the Lambda Function
+        """
+        non_existent_subnet_id = f"subnet-{short_uid()}"
+        wrong_format_subnet_id = f"bad-format-{short_uid()}"
+
+        # AWS validates the Security Group first, so we need a valid one to test SubnetsIds
+        security_groups = aws_client.ec2.describe_security_groups(MaxResults=5)["SecurityGroups"]
```

Comment:
I guess this just assumes that there is at least one valid security group (probably a sensible assumption)

Reply:
Yes, I think I've seen other tests doing this assumption with the default VPC, so figured I could as it'd be easier than to create the full VPC. Hopefully this is "safe" to do 😅